### ISSUE TITLE
📋 Daily Briefing — 2026-06-04

### ISSUE THREAD

**Issue by tardis-key** [tardis-key/codex#4]
# 📋 Daily Briefing — 2026-06-04

## 🚗 车辆提醒

> **保险** — 日期为到期日 ｜ **保养** — 日期为上次执行日，按行业周期推算下次时间

### 📊 车辆总览

| 车辆 | 状态 |
|------|------|
| 巧格 | 🟡 交强险人保2026.6.29 到期（25天后） |
| 甲壳虫 | 🟡 10.7wkm保养 2025.7.1（27天后） |
| 水鸟 | 🟡 交强险人保 2026.6.27 到期（23天后） |
| 幼兽 | ✅ 正常 |

### 🟡 即将到期 (30天内)

| 类型 | 车辆 | 事项 | 记录日期 | 到期日 | 剩余 |
|:----:|------|------|:----------:|:----:|:----:|
| 🛡️ | 水鸟 | 交强险人保 2026.6.27 到期 | 2026-06-27 | 2026-06-27 | 23 天 |
| 🛡️ | 巧格 | 交强险人保2026.6.29 到期 | 2026-06-29 | 2026-06-29 | 25 天 |
| 🔧 | 甲壳虫 | 10.7wkm保养 2025.7.1 | 2025-07-01 | 2026-07-01 | 27 天 |

## 🔧 verl · PR (9) + Issues (2)

> 数据来源：[verl-project/verl](https://github.com/verl-project/verl) · 过去 24 小时

### Pull Requests

| # | 标题 | 作者 | 状态 |
|---|------|------|:----:|
| [#6601](https://github.com/verl-project/verl/pull/6601) | [misc] fix: revert run_qwen3_8b_fsdp_npu.sh | hustmf | 🟢 open |
| [#6600](https://github.com/verl-project/verl/pull/6600) | [ci] fix: chang nightly ci qwen3 8b | daikang6 | 🟢 open |
| [#6598](https://github.com/verl-project/verl/pull/6598) | Full async sg lang weight broadcasting | theely | ⚫ closed |
| [#6597](https://github.com/verl-project/verl/pull/6597) | Sync fork | yaoliu13 | ⚫ closed |
| [#6596](https://github.com/verl-project/verl/pull/6596) | [megatron] fix: tolerate extra kwargs in MTP postprocess patch | Meirtz | 🟢 open |
| [#6595](https://github.com/verl-project/verl/pull/6595) | [data] fix: default image_patch_size to processor's real patch_size in filter_overlong_prompts | zengxingchen | 🟢 open |
| [#6594](https://github.com/verl-project/verl/pull/6594) | fix: align VL prompt filtering patch size | he-yufeng | 🟢 open |
| [#6593](https://github.com/verl-project/verl/pull/6593) | feat(distillation): chunked gather-logsumexp for top-K loss to avoid OOM at long context | kekellllll | 🟢 open |
| [#6591](https://github.com/verl-project/verl/pull/6591) | [vllm, megatron] fix: mxfp8 trainning support on Ascend NPU for verl v0.8.0 | quancs | 🟢 open |

### Issues

| # | 标题 | 作者 | 状态 |
|---|------|------|:----:|
| [#6599](https://github.com/verl-project/verl/issues/6599) | verl 微调多模态的数据格式为什么images不支持传路径呢，而只能传PIL.mage? | cqray1990 | 🟢 open |
| [#6592](https://github.com/verl-project/verl/issues/6592) | filter_overlong_prompts uses wrong image_patch_size (14) for VL models with patch_size=16, causing torch.cat crash in agent_loop | zengxingchen | 🟢 open |

---

<sub>🤖 由 **Codex** 自动生成 · 2026-06-04 11:04 · 非人工发布</sub>
